src/EditorFeatures/CSharpTest/MakeRefStruct/MakeRefStructTests.cs

```diff
@@ -0,0 +1,340 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Immutable;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.MakeRefStruct;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Diagnostics;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.MakeRefStruct
+{
+    public class MakeRefStructTests : AbstractCSharpDiagnosticProviderBasedUserDiagnosticTest
+    {
+        private static readonly CSharpParseOptions s_parseOptions =
+            CSharpParseOptions.Default.WithLanguageVersion(LanguageVersion.CSharp7_3);
+
+        internal override (DiagnosticAnalyzer, CodeFixProvider) CreateDiagnosticProviderAndFixer(Workspace workspace)
+            => (null, new MakeRefStructCodeFixProvider());
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsMakeRefStruct)]
+        [WorkItem(31831, "https://github.com/dotnet/roslyn/issues/31831")]
```

Comment:
@dotnet/roslyn-ide Do we use `WorkItemAttribute` for new features? I thought it only applies to tests for bug fixes.